tests/typescript/tests/example.spec.ts

```diff
@@ -0,0 +1,7 @@
+describe('saucectl demo test', () => {
+    test('verify ts is transpiling', async function () {
+        // Just transpile a typescript variable to confirm that it's transpiling typescript
+        const str:String = "Hello World";
```

Comment:
Strings are usually defined with lowercase ([#ref](https://www.typescriptlang.org/docs/handbook/basic-types.html#string))

```suggestion
        const str:string = "Hello World";
```